OSRMTextInstructions.podspec

```diff
@@ -48,10 +48,10 @@ Pod::Spec.new do |s|
 
   s.dependency "MapboxDirections.swift"
 
+  s.prepare_command = "./json2plist.sh"
```

Comment:
To be clear, the only files that changed in this PR are README.md and .travis.yml. The other files were being shown in the diff because of #16, which is now merged.